src/npc.cpp

```diff
@@ -2124,9 +2124,6 @@ void npc::shop_restock()
             }
         }
     }
-    if( value_groups.empty() && rigid_groups.empty() ) {
-        return;
-    }
 
```

Comment:
What's this change for?

Reply:
It's from a comment from andrei. See here. https://discord.com/channels/598523535169945603/598529174302490644/1003540642825257050